### PR TITLE
ncl Chapter 14

### PR DESCRIPTION
**Pull request by NormanLittle**

### CODE REVIEW

**Comment by alexdeas** [comment]
There was a break this week in case you're wondering where all the other pull requests are.

**Comment by NormanLittle** [comment]
Ah, thanks for letting me know @alexdeas.

File: answers.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+16) See [CommonFactor.java](src/com/selftest/CommonFactor.java)
+
+
+17)  
+```java
+SomeClass::new;
+```
+
+18)  java.util.function.
```
Comment (utopia-2) [inline answers.md:56]: :+1: all good.

File: src/com/selftest/CommonFactor.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		return value;
+	}
+
+	boolean isFactor( int number ) {
+		return value % number == 0;
+	}
+
+	boolean hasCommonFactor( int number ) {
+		return IntStream.range( 2, value )
+		                .anyMatch( i -> isFactor( i ) && number % i == 0 );
```
Comment (utopia-2) [inline src/com/selftest/CommonFactor.java:26]: A bit inefficient, but it does look elegant enough.

**Comment by SandyThomson** [comment]
@NormanLittle - just FYI, we had a vote and are skipping the applets part of the next chapter (and just do the 'Misc Topics right at the end). As filler, we are planning to mess around with lambdas a little more.

We plan to do Chapters 16 and 17 though.

**Pull request closed without merging**